PhotoCollection/PhotoCollection/PhotoController.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  PhotoController.swift
+//  PhotoCollection
+//
+//  Created by Cora Jacobson on 6/10/20.
+//  Copyright © 2020 Cora Jacobson. All rights reserved.
+//
+
+import Foundation
+
+class PhotoController {
```

Comment:
Very nice job with this class. The code is nice and clean, easy to follow. Everything functions exactly as it should.